prdoc/schema_user.json

```
@@ -0,0 +1,223 @@
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
+      },
+
+      "doc": {
+        "type": "array",
+        "description": "Description of the PR. Provide a description for each relevant audience.\nSee the `audience` property for more documentation about audiences",
+        "items": {
+          "$ref": "#/$defs/doc"
+        },
+        "minItems": 1
+      },
+      "crates": {
+        "description": "You have the option to provide input about the semver level of your changes for each crate\n. This is used during the crate publishing to crates.io and to help users understand the impact of the changes introduced in your PR.\nYou can also set it to `no change` if you know there was no significant semver change",
+        "type": "array",
+        "items": {
+          "$ref": "#/$defs/crate"
+        }
+      },
+      "migrations": {
+        "description": "It is important for users to be aware of migrations.\nMake sure to mention any migrations in the appropriate sub-properties",
+        "type": "object",
+        "properties": {
+          "db": {
+            "type": "array",
+            "nullable": false,
+            "description": "List of the Database Migrations or []",
+            "items": {
+              "$ref": "#/$defs/migration_db"
+            },
+            "minItems": 0,
+            "required": [
+              "name",
+              "description"
+            ]
+          },
+          "runtime": {
+            "type": "array",
+            "nullable": false,
+            "description": "List of the Runtime Migrations or []",
+            "minItems": 0,
+            "items": {
+              "$ref": "#/$defs/migration_runtime"
+            },
+            "required": [
+              "db",
+              "runtime"
+            ]
+          }
+
+        }
+      },
+      "host_functions": {
+        "description": "Lits of the host functions involved in this PR.\nThis is used to generate the host functions documentation.",
+        "type": "array",
+        "minItems": 0,
+        "items": {
+          "$ref": "#/$defs/host_function"
+        }
+      }
+    },
+    "required": [
+      "title",
+      "doc",
+      "crates",
+      "migrations",
+      "host_functions"
+    ],
+    "$defs": {
+      "audience": {
+        "oneOf": [
+          {"const": "Node Dev",
+          "title": "Those who build around the client side code. Alternative client builders, SMOLDOT, those who consume RPCs. These are people who are oblivious to the runtime changes. They only care about the meta-protocol, not the protocol itself."},
+
+          {"const": "Runtime Dev",
+          "title": "All of those who rely on the runtime. A parachain team that is using a pallet. A DApp that is using a pallet. These are people who care about the protocol (WASM), not he meta-protocol (client)."},
```

Small typo he -> the
```suggestion
          "title": "All of those who rely on the runtime. A parachain team that is using a pallet. A DApp that is using a pallet. These are people who care about the protocol (WASM), not the meta-protocol (client)."},
```